shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -81,26 +84,36 @@
       "workers.ssh.reachable",
       new SshValidationTask("workers"));
 
+  // space validations
+  private static final ValidationTask WORKER_RAMDISK_SPACE_VALIDATION_TASK = registerTask(
+      "worker.ramdisk.space",
+      new RamDiskSpaceValidationTask());
+
   private static final Map<String, Collection<ValidationTask>> TARGET_TASKS =
       initializeTargetTasks();
 
   private static Map<String, Collection<ValidationTask>> initializeTargetTasks() {
     Map<String, Collection<ValidationTask>> targetMap = new TreeMap<>();
-    targetMap.put("master", Arrays.asList(
-        MASTER_RPC_VALIDATION_TASK,
-        MASTER_WEB_VALIDATION_TASK,
-        PROXY_WEB_VALIDATION_TASK,
+    ValidationTask[] commonTasks = {
         MASTERS_SSH_VALIDATION_TASK,
+        PROXY_WEB_VALIDATION_TASK,
         WORKERS_SSH_VALIDATION_TASK
```

Comment:
I guess the constant name is kind of confusing here. `WORKERS_SSH_VALIDATION_TASK` checks that all nodes specified in `conf/workers` are reachable through ssh from the current node. `MASTERS_SSH_VALIDATION_TASK`checks that all nodes specified in `conf/masters` are reachable through ssh from the current node. We would like to check all nodes are reachable from all nodes therefore they are included as common tasks. 

How about I rename them to `SSH_TO_WORKERS_VALIDATION_TASK` and `SSH_TO_MASTERS_VALIDATION_TASK`?